docs/games/sm63/sm63.md

```diff
@@ -1,3 +1,15 @@
 # Super Mario 63
 
-Super Mario 63, or SM63 for short, is a fan-game by [Runouw](https://runouw.com) based on [Super Mario 64](https://wikipedia.org/wiki/Super_Mario_64), [Super Mario Sunshine](https://wikipedia.org/wiki/Super_Mario_Sunshine), and [Super Mario Galaxy](https://wikipedia.org/wiki/Super_Mario_Galaxy).
\ No newline at end of file
+Super Mario 63, or SM63 for short, is a fan-game by [Runouw](https://runouw.com) based on [Super Mario 64](https://wikipedia.org/wiki/Super_Mario_64), [Super Mario Sunshine](https://wikipedia.org/wiki/Super_Mario_Sunshine), and [Super Mario Galaxy](https://wikipedia.org/wiki/Super_Mario_Galaxy).
```

Comment:
Why is this line showing up as changed? What changed?

Reply:
good question

Reply:
Did you just accidentally add a space at the end? Is it some encoding thing? Either way, it's probably not important.